src/components/ChatForm/ToolUseSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, SegmentedControl, Text } from "@radix-ui/themes";
+import { ToolUse } from "../../features/Chat";
+
+type ToolUseSwitchProps = {
+  toolUse: ToolUse;
+  setToolUse: (toolUse: ToolUse) => void;
+};
+
+export const ToolUseSwitch = ({ toolUse, setToolUse }: ToolUseSwitchProps) => {
+  return (
+    <Flex direction="column" gap="8px" align="start">
+      <Text size="2">How fast you want the answer:</Text>
```

Comment:
Grammar,  `How fast to answer:` ? `How fast do you want the answer:` 

Also, I think `gap` should just be a number with out the `px` suffix and it'll apply one of the `var(--space-` variables. I think "2" is about 8px